### ISSUE TITLE
Windows release

### ISSUE THREAD

**Issue by littleli** [wilkerlucio/pathom-viz#34]
- Feature request :)
- There are Linux and Mac OS artifacts
- Explore the ways to build Windows artifacts too

**Comment by littleli** [comment]
![pathom-viz](https://user-images.githubusercontent.com/544082/111972673-fe230900-8afd-11eb-9787-d00663b38dc7.png)

- it turned out to be entirely possible
- scripts don't work well on Windows

**Comment by wilkerlucio** [comment]
Maybe that can work, the reason I didn't had a windows build was because the Clojure support for Windows on Github Actions wasn't there, but I see https://github.com/DeLaGuardo/setup-clojure/issues/1 and seems like its merged. I can try giving it another go 👍

**Comment by wilkerlucio** [comment]
I never built on Windows myself, do you have any comments about the process? Some hiccups you were able to overcome?

**Comment by littleli** [comment]
I recommend is to run everything from npm scripts instead of having shell (bash) scripts. It usually works everywhere including CI services and it unifies the build process.  

I tried instructions from the [readme](https://github.com/wilkerlucio/pathom-viz#13-creating-electron-app) but I had to slightly modify the steps on Windows:
```sh
npx shadow-cljs watch workspaces electron-background electron-renderer -A:dev:dev-ws:electron
cd shell\electron
npm i
npm run pack
```
It build without any issues. I have Clojure on my Windows box installed using [scoop installer](https://github.com/littleli/scoop-clojure). 

Also some Clojurists use appveyor to build Clojure stuff for Windows. I do use appveyor for my scoop project, but to honest I  don't have enough knowledge about this. There is an [example](https://www.electron.build/multi-platform-build#sample-appveyoryml-to-build-electron-app-for-windows) in electron-builder project.

**Comment by littleli** [comment]
Also I have a beef in this. If you're successful with the `portable` build I would like to add it to my scoop repository mentioned above.

**Comment by wilkerlucio** [comment]
Hello, I'm currently without a computer so its hard for me to try. Do you want to try it on your end? I think if you fork you can change the actions and run from your repo, if you can send a PR with the needed changes, I'll be glad to merge it in.

**Comment by wilkerlucio** [comment]
I just gave a try to that again today, but without success. This is where I got stuck: https://github.com/wilkerlucio/pathom-viz/runs/2397010521?check_suite_focus=true

The Clojure mode seems to support windows, but when I try to use it from Shadow there it fails.

**Comment by littleli** [comment]
hm, it seems to me that `clojure` is not available, right?

Can you try to put another step in the github action flow after clojure install step? Something like:

```
- name: Verify clojure installation
  run: clj --version
```

**Comment by wilkerlucio** [comment]
I'm not trying on this for the moment, but you can clone the repo and try yourself, if you figure it out I'll be happy to merge back.

**Comment by littleli** [comment]
Github action update PR: https://github.com/wilkerlucio/pathom-viz/pull/37

**Comment by littleli** [comment]
also Windows release now available via Scoop installer: https://github.com/littleli/scoop-clojure/pull/155

**Comment by wilkerlucio** [comment]
Closed by #37
